### PR TITLE
chore(release): v3.20.1-alpha.0

### PR DESCRIPTION
**Pull request by sfoster1**
First alpha for 3.20.1, including the fix for #6394

### CODE REVIEW

File: api/release-notes.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Robot OS Changes from 3.20.0 to 3.20.1
+
+## Bugfixes
+- Fix an issue introduced in 3.20.1 where protocol uploads might fail if the robot had been upgraded from Robot Software version 3.18.0 or previous ([#6394](https://github.com/Opentrons/opentrons/issues/6394))
```
Comment (b-cooper) [inline api/release-notes.md:4]: Should this say "Fix an issue introduced in 3.20.0"?

**Review by Laura-Danielle — APPROVED** [review]
Looks good aside from Brian's comment

File: api/release-notes.md (hunk 1 above)
Comment (sfoster1) [reply api/release-notes.md:4]: YEs

Comment (sfoster1) [inline api/release-notes.md:4]:
```suggestion
- Fix an issue introduced in 3.20.0 where protocol uploads might fail if the robot had been upgraded from Robot Software version 3.18.0 or previous ([#6394](https://github.com/Opentrons/opentrons/issues/6394))
```

**Pull request merged**